_component_design/help-beacon.md

```diff
@@ -4,13 +4,12 @@ parent: Help
 secondary: Notifications
 layout: component
 category: Components
-usage: |
-  These tooltips provide users with additional information while completing
-  tasks and alert them to new features.
-preview-image: components/preview-help.svg
-status: deprecated, in-progress
+usage: >
+  The Beacon can be used to identify the location of products or features in the control panel. It can also be used to indicate advanced functionality to existing features
```

Comment:
Nit: The capitalization of the element changes from file to file. I'd lowercase them all, so here it would be `beacon` instead of `Beacon`.

Reply:
Fixed in commit fe6a7cf and added to this PR.